docs/blog/author.yaml

```diff
@@ -312,6 +312,10 @@
   bio: Web Developer, Open Source advocate, new tech explorer - https://lannonbr.com/
   avatar: avatars/benjamin-lannon.jpg
   twitter: "@lannonbr"
+- id: Thomas Wang
```

Comment:
We can leave this in, but then we'll need a Gatsby blog post from you later on. Deal? 😄

Reply:
Deal :)